dev-games/irrlicht-headers/irrlicht-headers-1.8.4.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils
+
+MY_PN="irrlicht"
+MY_P="${MY_PN}-${PV}"
+DESCRIPTION="Irrlicht 3D engine headers"
+HOMEPAGE="http://irrlicht.sourceforge.net/"
+SRC_URI="mirror://sourceforge/irrlicht/${MY_P}.zip
+	https://dev.gentoo.org/~mgorny/dist/${MY_P}-patchset.tar.bz2"
+
+LICENSE="ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="app-arch/unzip"
```

Comment:
This package will need an `RDEPEND` blocker against older dev-games/irrlicht versions to prevent file conflicts.

Reply:
Would `RDEPEND="!<dev-games/irrlicht-1.8.4-r1"` be the correct syntax?